oh_staff_ui/views_utils.py

```diff
@@ -214,9 +216,25 @@ def get_edit_item_context(item_id: int) -> dict:
         "resource_formset": resource_formset,
         "subject_formset": subject_formset,
         "relatives": relatives,
+        "public_site_url": get_public_site_url(item),
     }
 
 
+def get_public_site_url(item: ProjectItem) -> str:
+    if not (
+        settings.OH_PUBLIC_SITE
+        and item.status.status in ("Completed", "Completed with minimal metadata")
+    ):
+        return None
+
+    if item.type.type == "Series":
+        return f"{settings.OH_PUBLIC_SITE}/?f%5Bseries_facet%5D%5B%5D={urllib.parse.quote_plus(item.title)}"
```

Comment:
Instead of building a pre-encoded/escaped URL here, I prefer the simpler and self-documenting:

`return f"{settings.OH_PUBLIC_SITE}/?f[series_facet][]={item.title}"`

This string is being passed to a Django template, which can escape/encode URLs when needed (though I tested, and this doesn't seem to need it).  Please try it yourself, and if you agree, let's use this simpler non-encoded version.

If you do want to keep the pre-encoded version, please document it via a comment above that line, so readers know what's been encoded.